packages/cli/src/commands/generate/scaffold/scaffold.js

```diff
@@ -221,25 +313,42 @@ const addScaffoldImport = () => {
   return 'Added scaffold import to index.js'
 }
 
-export const command = 'scaffold <model>'
+export const command = 'scaffold <pathSlashModel>'
 export const desc = 'Generate pages, SDL, and a services object.'
 export const builder = {
   force: { type: 'boolean', default: false },
+  // So the user can specify a path to nest the generated files under.
+  // E.g. yarn rw g scaffold post --path=admin
+  path: { type: 'string', default: false },
 }
-export const handler = async ({ model, force }) => {
+// The user can also specify a path in the argument.
+// E.g. yarn rw g scaffold admin/post
+export const handler = async ({ pathSlashModel, force, path: pathFlag }) => {
```

Comment:
possible to add a Yargs 'description' key in here? It won't display correctly yet, e.g. `yarn rw g scaffold --help` currently shows help for `yarn rw g` only. But it helps code now and will be available via CLI in future once we re-structure Yargs:
https://yargs.js.org/docs/#api-optionskey-opt

Reply:
For the path flag, how does `path to put generated files in` sound? Alternatives:
- `path to generated files`
- `path to nest generated files under`
- `directory(ies) to nest generated files under`
  
And the command's description could change from/to `Generates pages, SDL, and a services object` => `Generates and optionally nests pages, SDL, and a services object`